lib/rocketfuel_api/service.rb

```diff
@@ -19,12 +19,13 @@ def name
     end
   end
 
-  def plural_name
-    name + 's'
-  end
-
   def uri_suffix
-    plural_name
+    file_path = RocketfuelApi.root.join('lib', 'config', 'endpoints_for_services.yaml')
+    endpoints = YAML.load_file(file_path)
```

Comment:
Great! Would be good if we could somehow cache the endpoints hash so we don't need to load yml multiple times if we are using more than 1 service.